src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Icon } from "@chakra-ui/react"
+import TooltipComponent from "."
+
+type TooltipType = typeof TooltipComponent
+
+const meta: Meta<TooltipType> = {
+  title: "Molecules / Overlay Content / Tooltip",
+  component: TooltipComponent,
+}
+
+export default meta
+
+type Story = StoryObj<TooltipType>
+
+export const Tooltip: Story = {
+  render: () => (
+    <TooltipComponent content="Text here">
+      <Icon fill="'text" name="info" />
```

Comment:
This should render a specific component, and just use the `Icon` component. Probably...
```suggestion
      <Icon as={ RiInformationLine } />
```

No need to provide a `name` here. And because Chakra's `Icon` component automatically renders `fill="currentColor"` then any color should be established from the parent only via the `color` prop. (Especially if any text or icon rendered were to use the same color styles anyway!)